src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.CinemaHallRequestDto;
+import mate.academy.spring.dto.CinemaHallResponseDto;
+import mate.academy.spring.service.CinemaHallService;
+import mate.academy.spring.service.mapper.CinemaHallDtoMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallDtoMapper cinemaHallDtoMapper;
+    private final CinemaHallService cinemaHallService;
+
+    public CinemaHallController(CinemaHallDtoMapper cinemaHallDtoMapper,
+                                CinemaHallService cinemaHallService) {
+        this.cinemaHallDtoMapper = cinemaHallDtoMapper;
+        this.cinemaHallService = cinemaHallService;
+    }
+
+    @GetMapping
+    public List<CinemaHallResponseDto> getAll() {
+        return cinemaHallService.getAll()
+                .stream()
+                .map(m -> cinemaHallDtoMapper.toDto(m))
```

Comment:
```suggestion
                .map(cinemaHallDtoMapper::toDto)
```
not critical, just a suggestion